service/src/parentchain_handler.rs

```diff
@@ -227,6 +227,8 @@ where
 		while last_synced_header.number() < until_header.number() {
 			last_synced_header = self.sync_parentchain(last_synced_header)?;
 			trace!("[{:?}] synced block number: {}", id, last_synced_header.number);
+			// fast polling is useless because we only loop here if the target block hasn't been finalized yet
```

Comment:
this can only be a temporary workaround to gracefully wait until register_sgx_eclave block got finalized

Reply:
done